packages/governance/src/votes/votes.cairo

```diff
@@ -156,19 +200,46 @@ pub mod VotesComponent {
     }
 
     //
-    // Internal
+    // Internal for ERC20Votes
+    //
```

Comment:
I think we can remove these comments for simplicity. Maybe a single comment for all the implementations works better, but none is looks ok to me.